crates/uv-resolver/src/lock/mod.rs

```diff
@@ -4691,7 +4732,7 @@ fn normalize_requirement(
                 name: requirement.name,
                 extras: requirement.extras,
                 groups: requirement.groups,
-                marker: requirement.marker,
+                marker: requires_python.simplify_markers(requirement.marker),
```

Comment:
I'm a little confused as to whether this should be simplify or complexify. It might not matter as long as it's consistent?

Reply:
Yeah that sounds right to me.